### ISSUE TITLE
Crash when building env

### ISSUE THREAD

**Issue by rasmusagren** [QuantStack/mamba#59]
Hi,

I decided to try out mamba, but I get the following when I try to create an env.

```
(base) $ mamba env create -n osegra -f environment.yml
Getting  scilifelab-lts linux-64
Getting  scilifelab-lts noarch
Getting  conda-forge linux-64
Getting  conda-forge noarch
Getting  bioconda linux-64
Getting  bioconda noarch
Getting  pkgs/main linux-64
Getting  pkgs/main noarch
Getting  pkgs/r linux-64
Getting  pkgs/r noarch


Looking for: ['lts-workflows', 'bamtools', 'bedtools', 'biopython', 'multiqc', 'picard', 'pyyaml', 'yamlloader', 'rsem', 'star', 'ucsc-gtftogenepred', 'perl', 'r-rmarkdown', 'r-yaml', 'r-gplots', 'r-optparse', 'r-vioplot', 'subread', 'bioconductor-org.hs.eg.db', 'bioconductor-sc3', 'bioconductor-scran', 'snakemake-minimal', 'bioconductor-scater', 'bioconductor-biomart', 'r-mvoutlier', 'r-rmarkdown', 'r-rtsne', 'jupyter', 'r-irkernel', 'r-formatr']


8317 packages in https://conda.anaconda.org/bioconda/noarch
27095 packages in https://conda.anaconda.org/bioconda/linux-64
16944 packages in https://conda.anaconda.org/conda-forge/noarch
74033 packages in https://conda.anaconda.org/conda-forge/linux-64
842 packages in https://repo.anaconda.com/pkgs/main/noarch
5893 packages in https://repo.anaconda.com/pkgs/r/linux-64
14232 packages in https://repo.anaconda.com/pkgs/main/linux-64
3536 packages in https://repo.anaconda.com/pkgs/r/noarch
0 packages in https://conda.anaconda.org/scilifelab-lts/noarch
26 packages in https://conda.anaconda.org/scilifelab-lts/linux-64

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/khkk378/miniconda3/lib/python3.7/site-packages/conda/exceptions.py", line 1074, in __call__
        return func(*args, **kwargs)
      File "/home/khkk378/miniconda3/lib/python3.7/site-packages/conda_env/cli/main.py", line 78, in do_call
        exit_code = getattr(module, func_name)(args, parser)
      File "/home/khkk378/miniconda3/lib/python3.7/site-packages/conda_env/cli/main_create.py", line 102, in execute
        result[installer_type] = installer.install(prefix, pkg_specs, args, env)
      File "/home/khkk378/miniconda3/lib/python3.7/site-packages/mamba/mamba_env.py", line 108, in mamba_install
        update_specs  = specs
    TypeError: __new__() missing 1 required positional argument: 'neutered_specs'

`$ /home/khkk378/miniconda3/bin/mamba create -n osegra -f environment.yml`

  environment variables:
                 CIO_TEST=<not set>
  CONDA_AUTO_UPDATE_CONDA=false
        CONDA_DEFAULT_ENV=base
                CONDA_EXE=/home/khkk378/miniconda3/bin/conda
             CONDA_PREFIX=/home/khkk378/miniconda3
    CONDA_PROMPT_MODIFIER=(base)
         CONDA_PYTHON_EXE=/home/khkk378/miniconda3/bin/python
               CONDA_ROOT=/home/khkk378/miniconda3
              CONDA_SHLVL=1
                FTP_PROXY=<set>
              HTTPS_PROXY=<set>
               HTTP_PROXY=<set>
                  MANPATH=/usr/share/lmod/lmod/share/man:
               MODULEPATH=/opt/scp/modules/all:/usr/share/modulefiles/Linux:/usr/share/modulefil
                          es/Core:/usr/share/lmod/lmod/modulefiles/Core
                 NO_PROXY=<set>
                     PATH=/home/khkk378/miniconda3/bin:/home/khkk378/miniconda3/condabin:/usr/li
                          b64/qt-3.3/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/opt/
                          ibutils/bin:/opt/dell/srvadmin/bin:/home/khkk378/miniconda3/bin:/home/
                          khkk378/.local/bin:/home/khkk378/bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>
                ftp_proxy=<set>
               http_proxy=<set>
              https_proxy=<set>
                 no_proxy=<set>

     active environment : base
    active env location : /home/khkk378/miniconda3
            shell level : 1
       user config file : /home/khkk378/.condarc
 populated config files : /home/khkk378/.condarc
          conda version : 4.7.12
    conda-build version : not installed
         python version : 3.7.3.final.0
       virtual packages :
       base environment : /home/khkk378/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/khkk378/miniconda3/pkgs
                          /home/khkk378/.conda/pkgs
       envs directories : /home/khkk378/miniconda3/envs
                          /home/khkk378/.conda/envs
               platform : linux-64
             user-agent : conda/4.7.12 requests/2.22.0 CPython/3.7.3 Linux/3.10.0-957.27.2.el7.x86_64 centos/7.6.1810 glibc/2.17
                UID:GID : 33392639:408929969
             netrc file : /home/khkk378/.netrc
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.
```

**Comment by wolfv** [comment]
are you on windows, linux or os x?

Can you check that you have the most recent version of mamba (0.0.9?)
this issue was fixed for linux and os x a couple of days ago.

**Comment by rasmusagren** [comment]
I'm on Linux and I just installed it for the first time before running this. I have 0.0.9 from conda-forge.

**Comment by wolfv** [comment]
ok, indeed we forgot to fix one line for the environment creation from spec. I am pushing a fix now, and it will go into 0.0.10 in a couple of minutes. sorry about that. 

you can fix it locally by going to the place where `mamba` is installed, search for `mamba_env.py` and add `neutered_specs` like so

```
    pref_setup = PrefixSetup(
        target_prefix = prefix,
        unlink_precs  = unlink_precs,
        link_precs    = link_precs,
        remove_specs  = [],
        update_specs  = specs,
        neutered_specs = ()
    )
```

**Comment by wolfv** [comment]
@rasmusagren the fix was merged, I am releasing 0.0.10 to conda-forge right now but it will take a bit of time to propagate to the channel (https://github.com/conda-forge/mamba-feedstock/pull/18).

Thanks for the bug report. Feel free to let me know how mamba works for you!

**Comment by rasmusagren** [comment]
Thanks a lot!
